README.md

```diff
@@ -35,4 +35,4 @@
     <li>Instagram <a target="_blank" href="https://www.instagram.com/grand_alchemist/">here</a>.</li>
 </ul>
 <h3>Contributions</h3>
-<p> Everyone is free to make sugestions. for major changes make a pull request
+<p> Everyone is free to make sugestions. for major changes make a pull request. Don't alter the maser branch unless you are Julian Carracedo or a member of Microverse team.</p>
```

Comment:
## Required changes
- [ ] In your next round of changes, you are expected to stick to this `Pull Request` and not open a new one. This will help the next reviewer make an informed decision and track changes made based on the events in this PR. 
- [ ]  Commit changes as much as possible as well in order to make it easier in tracking the changes made. 
##

###   Improve the `navbar` to match the source page.
- [ ] The `Hamburger button`, the `search icon`,  and the `link` with text should be on the leftmost side of the navbar. Kindly fix this.
- [ ] Ensure the `Login` elements are on the rightmost side of the navbar as well. Work on the colour topography as well
- [ ] Replace the current `main article image` of Microverse with the actual image from the source page.
- [ ] Ensure the clone page uses the same `fonts` as the source page. 
- [ ] Be mindful of the `social links` above the image, its `caption` and the `width` of the image as well.

Below is a comparison between the two navbars and the main article image  Please ensure the clone resembles the source page in everything as shown.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JuliCarracedo/NYTMimic/htmlAndCss/index.html)|
|:--:|:--:|
|![main-article](https://user-images.githubusercontent.com/22345961/112228966-28321380-8c43-11eb-91ad-2e429af0b9ce.png)|![main-article-clone](https://user-images.githubusercontent.com/22345961/112228980-32eca880-8c43-11eb-91ba-9967389dcbec.png)|



##

### Improve the Editor's pick section and the layout of the main content.
- [ ] The layout of the text and images should be well centred. Please fix this.

Below I have shared some inspiration

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JuliCarracedo/NYTMimic/htmlAndCss/index.html)|
|:--:|:--:|
|![editors-pick](https://user-images.githubusercontent.com/22345961/112230005-20736e80-8c45-11eb-89ae-68997377550c.png)|![editors-pick-clone](https://user-images.githubusercontent.com/22345961/112230015-2701e600-8c45-11eb-943e-14b91bb165b6.png)|

##

Improve the ` The Theory of Inflation` section

- [ ] Good job implementing this section with  `grid`.The section should however meet the set standards of having its details sections implemented fully.  All the images and texts in this section are mandatory and ought to be implemented.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JuliCarracedo/NYTMimic/htmlAndCss/index.html)|
|:--:|:--:|
|![theory-of-inlation](https://user-images.githubusercontent.com/22345961/112230463-e6ef3300-8c45-11eb-9497-e3a24a4d0a05.png)|![theory-of-inlation-clone](https://user-images.githubusercontent.com/22345961/112230482-f1113180-8c45-11eb-8325-3682ca24c638.png)|

##

### Improve the `footer section`
The footer is expected to have:
- [ ] All the contents in place and well-styled.
- [ ] All the missing sections implemented including the `Read 365 comments` sections together with the accompanied social icons.
#### OPTIONAL: 
* [ ] It has a fixed footer (it sticks to the bottom part of the screen) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

> 


|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/JuliCarracedo/NYTMimic/htmlAndCss/index.html)|
|:--:|:--:|
|![footer](https://user-images.githubusercontent.com/22345961/112230881-bc51aa00-8c46-11eb-9164-666ff2b9e286.png)|![footer-clone](https://user-images.githubusercontent.com/22345961/112230901-c2478b00-8c46-11eb-86b6-acafb91b8080.png)|

 ### Implement missing sections 
The following sections have not been implemented on the clone page. I have highlighted them below for easier reference. 
#### Suggested newsletters section

![Suugessted-newsletters-section](https://user-images.githubusercontent.com/22345961/112230996-edca7580-8c46-11eb-9800-27f2b2382de9.png)

#### More into space and astronomy

![more-inspace-and-astronomy](https://user-images.githubusercontent.com/22345961/112231051-020e7280-8c47-11eb-83f2-f4bf04221c9c.png)

#### Editors pick

![editors-pick-footer](https://user-images.githubusercontent.com/22345961/112231077-12265200-8c47-11eb-9ef8-40d723e0d913.png)

## Improve the README
- [ ] Inline with having a descriptive README, do you mind adding an alive link to where the page is hosted live, and a screenshot of the page as well? 

You can use this [template ](https://github.com/microverseinc/readme-template)from [pathwright ](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)for inspiration. You are also welcomed to use this [tool ](https://raw.githack.com/)to generate a live link.